### ISSUE TITLE
Include hotkeys in RD menu for enabled commands

### ISSUE THREAD

**Issue by IvenBach** [rubberduck-vba/Rubberduck#3674]
Looking at #3592 I had to remember what the hotkey was. If this were listed for each command it would help alert users to the hotkey functionality.
![untitled](https://user-images.githubusercontent.com/26076874/34623520-88d1ca08-f206-11e7-8ddf-3c6cfeb14ae9.png)

**Comment by Vogel612** [comment]
This would mean that CommandMenuItems suddenly need to know about Hotkeys and be notified of their changes. 

I like the feature, and it's conceptually not that hard, but IIRC it would require a few deep architectural changes, that's why I've labeled it difficulty-03

**Comment by MDoerner** [comment]
I do not think that the commands need to know the hotkeys. (We just removed that knowledge.) It would suffice if the `CommandMenuItem`s knew them.

**Comment by retailcoder** [comment]
The code for this is already there, *something* broke it.

https://github.com/rubberduck-vba/Rubberduck/blob/28dcb9d0c8511fec537ad1c260db0d429f659eb7/RetailCoder.VBE/UI/Command/MenuItems/ParentMenus/ParentMenuItemBase.cs#L62
